internal/graphql/resolver_property.go

```diff
@@ -465,12 +465,12 @@ func (*propertyGroupResolver) SchemaGroup(ctx context.Context, obj *graphql1.Pro
 	return s.Group(obj.SchemaGroupID), nil
 }
 
-func actualValue(datasetLoader dataloader.DatasetDataLoader, value interface{}, links []*graphql1.PropertyFieldLink) (interface{}, error) {
-	if len(links) == 0 {
+func actualValue(datasetLoader dataloader.DatasetDataLoader, value interface{}, links []*graphql1.PropertyFieldLink, overridden bool) (interface{}, error) {
+	if len(links) == 0 || overridden {
 		return &value, nil
 	}
 	// 先頭のリンクにしかDatasetが割り当てられていない→先頭から順々に辿っていく
-	if len(links) > 1 && links[0].DatasetID != nil && links[len(links)-1].DatasetID == nil {
+	if len(links) >= 1 && links[0].DatasetID != nil && links[len(links)-1].DatasetID == nil {
```

Comment:
If there was only one link, `else` block would be executed and it should work. Did you change this because it didn't work?

Reply:
Sorry about this, It seems I forgot it while debugging